kolibri/plugins/qti_viewer/assets/src/views/AssessmentItemRef.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+
+  <AssessmentItem
+    v-if="assessmentItem"
+    :dom="assessmentItem"
+    :qtiFile="qtiFile"
+    :filePath="filePath"
+  />
+
+</template>
+
+
+<script>
+
+  import domMixin from '../mixins/domMixin';
+  import AssessmentItem from './AssessmentItem';
+
+  export default {
+    name: 'AssessmentItemRef',
+    components: {
+      AssessmentItem,
+    },
+    mixins: [domMixin],
+    data() {
+      return {
+        assessmentItem: null,
+      };
+    },
+    computed: {
+      href() {
+        const relativeHref = this.dom && this.dom.getAttribute('href');
+        if (relativeHref) {
+          const path = new URL(
+            relativeHref,
+            new URL(this.filePath, 'http://b.b/')
```

Comment:
Where does b.b come from here?

Reply:
This is the easiest way to do URL path concatenation, by using a dummy origin and then reading the path off the resultant URL object.